src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/Cardstrings.json

```
@@ -0,0 +1,51 @@
+{
+  "${ModID}:Knighthood": {
+    "NAME": "Knighthood",
+    "DESCRIPTION": "Take !M! damage. NL Choose a card in hand to Upgrade permanently. NL Exhaust.",
+    "EXTENDED_DESCRIPTION": [
+      " to Upgrade permanently."
+    ]
+  },
+  "${ModID}:Inferno": {
+    "NAME": "Inferno",
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL If this did not end combat, take 3 damage."
+  },
+  "${ModID}:Minniegun": {
+    "NAME": "Minniegun",
+    "DESCRIPTION": "Deal !D! damage to a random enemy !M! times. NL Put a *Jam in your discard pile."
+  },
+  "${ModID}:Ouroboros": {
+    "NAME": "Ouroboros",
+    "DESCRIPTION": "At the end of any turn in which you fully blocked a source of damage, gain 1 Strength.",
```

Very minor, but this might read just slightly better if it said "any source of damage" instead of "a". I think it helps make the connection that even sources of damage like your own card or Beat of Death count.
```suggestion
    "DESCRIPTION": "At the end of any turn in which you fully blocked any source of damage, gain 1 Strength.",
```